bubo/config.py

```diff
@@ -121,6 +121,11 @@ def __init__(self, filepath):
         if self.email and self.email.get("starttls") and self.email.get("ssl"):
             raise ConfigError("Cannot enable both starttls and ssl for email")
 
+        # Pindora
+        self.pindora = self._get_cfg(["pindora"], default={}, required=False)
```

Comment:
As you are defining all the pindora config below separately, you don't need this one here.